src/switch_core_session.c

```diff
@@ -2537,6 +2540,7 @@ SWITCH_DECLARE(switch_core_session_t *) switch_core_session_request_uuid(switch_
 	switch_queue_create(&session->event_queue, SWITCH_EVENT_QUEUE_LEN, session->pool);
 	switch_queue_create(&session->private_event_queue, SWITCH_EVENT_QUEUE_LEN, session->pool);
 	switch_queue_create(&session->private_event_queue_pri, SWITCH_EVENT_QUEUE_LEN, session->pool);
+	switch_core_hash_init(&session->media_extensions);
 
 	switch_mutex_lock(runtime.session_hash_mutex);
 	switch_core_hash_insert(session_manager.session_table, session->uuid_str, session);
```

Comment:
You'll need to find extension by id from switch_rtp.c. This is not enough to put ext into SDP, you also need to keep track of id to ext mapping per RTP session. You'll need some `_find_by_id` sort of function @StefanYohansson 
You'll need to consider 1-byte and 2-byte extensions https://datatracker.ietf.org/doc/html/rfc8285